test/integration/setup.js

```diff
@@ -24,15 +24,15 @@ const request = require('request');
 before(function(done) {
 	const that = this;
 
-	that.baseUrl = 'http://localhost:' + config.port + '/';
-	that.app = null;
-	that.last = {};
-	that.navigate = createNavigator(that.baseUrl, that.last);
+	this.baseUrl = 'http://localhost:' + config.port + '/';
```

Comment:
could we use a template string here?

Reply:
We could indeed. Tbh we need a PR that goes through and generally updates the code (using fat-arrow, template-literals etc).
I'd prefer to do it all in one fell-swoop instead